scripts/create_test_customer.py

```diff
@@ -92,20 +98,43 @@ def create_provider(self):
 
 
 if __name__ == '__main__':
+    default_api_host = os.getenv('KOKU_HOST', 'localhost')
+    default_api_port = os.getenv('KOKU_PORT', '80')
+    default_api_admin = os.getenv('KOKU_ADMIN', 'admin')
+    default_api_pass = os.getenv('KOKU_PASSWORD', 'pass')
+
+    default_db_host = os.getenv('POSTGRES_SQL_SERVICE_HOST', 'localhost')
+    default_db_port = os.getenv('POSTGRES_SQL_SERVICE_PORT', '15432')
+    default_db_name = os.getenv('DATABASE_NAME', 'koku')
+    default_db_user = os.getenv('DATABASE_USER', 'postgres')
```

Comment:
If the postgres user ends up being used to create the schema and tables, and the postgres user is not the user koku or masu uses, this will cause permissions errors.

Reply:
Do you have a suggestion on what a better default value should be?

Reply:
I would say the default should be requiring the environment variable. It's required for the app to run, and should be the same user so permissions line up.